src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/SetCustomerFormSubscriber.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\EventSubscriber;
+namespace Sylius\Bundle\CoreBundle\Form\EventSubscriber;
```

Comment:
It assigns customer only if there is no corresponding user account.
But ok, there should be in CustomerBundle the default implementation with condition 
`if (null !== $customer)`